scripts/tests

```diff
@@ -113,10 +113,11 @@ def base_run(
     verbose,
     run_as_root=False,
     exec_wrapper=None,
+    cross_prefix="",
 ):
     """Run the binary in all different ways"""
     cmd = [f"./{bin}"]
-    if force_qemu or (platform.system() == "Linux" and platform.machine() == "x86_64"):
+    if force_qemu or cross_prefix.strip() != "":
```

Comment:
This was needed to stop `tests` using `qemu` unconditionally on x86_64. Now it's only using it if cross compilation is used.